<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowWidget</class>
 <widget class="QMainWindow" name="MainWindowWidget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>434</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>679</width>
    <height>434</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BudgetBeam</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>679</width>
      <height>434</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>679</width>
      <height>434</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="mainPage">
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>20</y>
        <width>690</width>
        <height>87</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="logo">
         <property name="maximumSize">
          <size>
           <width>139</width>
           <height>80</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>C:/Users/amank/Downloads/DALLÂ·E 2023-11-08 06.02.55 - Design a small, sleek logo for a company called BudgetBeam. The logo should feature an abstract, geometric representation of a beam of light, using sh (4).png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>JetBrainsMono Nerd Font Mono</family>
           <pointsize>16</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>BudgetBeam, the smart financial mind.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="line">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>651</width>
        <height>20</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::HLine</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>651</width>
        <height>221</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>JetBrainsMono Nerd Font Mono</family>
        <bold>true</bold>
       </font>
      </property>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-10</y>
         <width>661</width>
         <height>221</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="verticalSpacing">
         <number>0</number>
        </property>
        <item row="1" column="1">
         <widget class="QPushButton" name="expensesButton">
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>Expenses</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="settingsButton">
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>Settings</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="insightsButton">
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>Insights</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="3">
         <widget class="QPushButton" name="contactButton">
          <property name="maximumSize">
           <size>
            <width>1000</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>Contact Developers</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="ledgerButton">
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>Ledger</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="dashboardButton">
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>About Me</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="finspireButton">
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>Finspire</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="finspirePage">
     <widget class="QPushButton" name="previousButton">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>290</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Main Page</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>671</width>
        <height>201</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="quoteLabel">
         <property name="maximumSize">
          <size>
           <width>611</width>
           <height>171</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="contactPage">
     <widget class="QLabel" name="contactLabel">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>541</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>JetBrainsMono Nerd Font</family>
        <pointsize>24</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Contact the developers</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="Line" name="contactLine">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>100</y>
        <width>561</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="previousButtonContact">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>380</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Main Page</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>160</y>
        <width>521</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="contactPageLayout"/>
     </widget>
    </widget>
    <widget class="QWidget" name="expensesPage">
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>641</width>
        <height>211</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="expensesLayout">
       <item>
        <widget class="QTableWidget" name="expensesTable">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>341</width>
        <height>171</height>
       </rect>
      </property>
      <property name="title">
       <string>Add Entry</string>
      </property>
      <widget class="QWidget" name="formLayoutWidget">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>29</y>
         <width>321</width>
         <height>117</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="labelDate">
          <property name="text">
           <string>Date: </string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDateEdit" name="dateEdit">
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelAmount">
          <property name="text">
           <string>Amount: </string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="amountLineEdit"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelCategory">
          <property name="text">
           <string>Category: </string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="labelDesc">
          <property name="text">
           <string>Description: </string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="descriptionLineEdit"/>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="categoryComboBox">
          <item>
           <property name="text">
            <string>Dining</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Education</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Entertainment</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Groceries</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Health</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Housing</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Utilities</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Transportation</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Miscellaneous</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="saveButton">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>146</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
      <widget class="Line" name="line_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="Line" name="line_3">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>140</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>40</y>
        <width>82</width>
        <height>56</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QPushButton" name="expensesPreviousButton">
         <property name="text">
          <string>Main Page</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="insightsPage">
     <widget class="QPushButton" name="insightsPreviousButton">
      <property name="geometry">
       <rect>
        <x>569</x>
        <y>20</y>
        <width>71</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Main Page</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>611</width>
        <height>321</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
       <item>
        <widget class="QLabel" name="fixedOne">
         <property name="font">
          <font>
           <family>JetBrainsMono Nerd Font</family>
           <pointsize>22</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>What do you want to ask today? :)</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="insightsLabel">
         <property name="font">
          <font>
           <family>Source Code Pro</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>611</width>
        <height>46</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QCheckBox" name="morePower">
           <property name="font">
            <font>
             <pointsize>7</pointsize>
            </font>
           </property>
           <property name="text">
            <string>More Power</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="maxPower">
           <property name="font">
            <font>
             <pointsize>7</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Max Power</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="askButton">
         <property name="text">
          <string>Ask</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="refreshButton">
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="ledgerPage">
     <widget class="QLabel" name="ledgerLabel">
      <property name="geometry">
       <rect>
        <x>38</x>
        <y>30</y>
        <width>601</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>JetBrainsMonoNL Nerd Font</family>
        <pointsize>24</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Your current blockchain ledger</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="Line" name="line_4">
      <property name="geometry">
       <rect>
        <x>57</x>
        <y>100</y>
        <width>571</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>59</x>
        <y>160</y>
        <width>571</width>
        <height>241</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QTableWidget" name="ledgerTable"/>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="ledgerPreviousButton">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>10</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Main Page</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="settingsPage">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>591</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>JetBrainsMono NF</family>
        <pointsize>36</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Proceed with caution</string>
      </property>
     </widget>
     <widget class="Line" name="line_5">
      <property name="geometry">
       <rect>
        <x>57</x>
        <y>100</y>
        <width>581</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>195</y>
        <width>89</width>
        <height>91</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="deleteDataButton">
         <property name="text">
          <string>Delete my data</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="settingsPreviousButton">
         <property name="text">
          <string>Take me back</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="aboutMe">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>100</y>
        <width>231</width>
        <height>171</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/icons/icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>581</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>JetBrainsMonoNL Nerd Font</family>
        <pointsize>28</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>BudgetBeam</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>300</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>v0.0.1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>340</y>
        <width>441</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>JetBrainsMono Nerd Font</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Langchain ð LLMs</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>390</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
